msmtools/analysis/sparse/stationary_vector.py

```diff
@@ -140,18 +140,6 @@ def stationary_distribution(T):
         Vector of stationary probabilities.
 
     """
-    fallback = False
-    try:
-        mu = stationary_distribution_from_backward_iteration(T)
-        if np.any(mu < 0):  # numerical problem, fall back to more robust algorithm.
-            fallback=True
-    except RuntimeError:
-        fallback = True
-
-    if fallback:
-        mu = stationary_distribution_from_eigenvector(T)
-        if np.any(mu < 0):  # still? Then set to 0 and renormalize
-            mu = np.maximum(mu, 0.0)
-            mu /= mu.sum()
+    mu = stationary_distribution_from_eigenvector(T)
```

Comment:
This is fine from my side. I'm just not sure why the MSMTools guys used `stationary_distribution_from_backward_iteration` in the first place... I don't want to induce troubles for them, since we still intend to make a pull request to the original repo.
Any ideas?

Reply:
Puh, really don't know... In the end, `stationary_distribution_from_eigenvector` will use iterative methods as well to find that top eigenvector, it's just much more optimized. I see two possibilities: 
- merge now and discuss this with the MSMTool guys when we open PR to upstream repo
- include parameter `use_backwards_iter: bool = False`

Reply:
Ok, I vote for merging and sorting it out with them later... ;-)

Reply:
okay, cool, then it's just removing of scipy sorting and we can merge.